<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>GeneralPage</class>
 <widget class="QWidget" name="GeneralPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>522</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>5</hsizetype>
    <vsizetype>5</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Form1</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_6" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="1" colspan="2" >
       <widget class="QLineEdit" name="remoteName" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QSpinBox" name="remotePort" >
        <property name="maximum" >
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QSpinBox" name="localPort" >
        <property name="maximum" >
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="textLabel2_2" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>1</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Remote server:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="textLabel4_2" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>1</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Local port number:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="textLabel3_2" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>1</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Remote port number:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_5" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="4" column="2" colspan="2" >
       <widget class="QDoubleSpinBox" name="multipleConnectionsPenaltyDoubleSpinBox" >
        <property name="decimals" >
         <number>4</number>
        </property>
        <property name="maximum" >
         <double>100</double>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2" >
       <widget class="QDoubleSpinBox" name="correctPositionBonusDoubleSpinBox" >
        <property name="decimals" >
         <number>4</number>
        </property>
        <property name="maximum" >
         <double>100</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2" >
       <widget class="QLabel" name="label_7" >
        <property name="text" >
         <string>Multiple Connections Penalty:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Correct Position bonus:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2" >
       <widget class="QDoubleSpinBox" name="newRoomPenaltyDoubleSpinBox" >
        <property name="decimals" >
         <number>4</number>
        </property>
        <property name="maximum" >
         <double>100</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Accept Best Absolute:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>New Room Penalty:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" >
       <widget class="QSpinBox" name="maxPaths" >
        <property name="maximum" >
         <number>100000</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2" >
       <widget class="QDoubleSpinBox" name="acceptBestAbsoluteDoubleSpinBox" >
        <property name="decimals" >
         <number>4</number>
        </property>
        <property name="maximum" >
         <double>100</double>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2" >
       <widget class="QDoubleSpinBox" name="acceptBestRelativeDoubleSpinBox" >
        <property name="decimals" >
         <number>4</number>
        </property>
        <property name="maximum" >
         <double>100</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Accept Best Relative:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Max Paths:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" colspan="2" >
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>Matching Tolerance:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4" >
       <widget class="QSpinBox" name="matchingToleranceSpinBox" >
        <property name="maximum" >
         <number>100</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_4" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="2" column="1" >
       <widget class="QCheckBox" name="emulatedExits" >
        <property name="text" >
         <string>Show emulated exits</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QCheckBox" name="drawUpperLayersTextured" >
        <property name="text" >
         <string>Draw upper layers textured</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="updated" >
        <property name="text" >
         <string>Show non updated rooms</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QCheckBox" name="drawNotMappedExits" >
        <property name="text" >
         <string>Draw not mapped exits</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QCheckBox" name="brief" >
        <property name="text" >
         <string>Brief mode</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="2" >
       <widget class="QPushButton" name="sellectLogFileButton" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>0</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>45</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>45</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text" >
         <string>Select</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="autoLoadFileName" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="logFileName" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QPushButton" name="sellectWorldFileButton" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>0</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>45</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>45</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text" >
         <string>Select</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QCheckBox" name="autoLoadCheck" >
        <property name="text" >
         <string>Auto load world:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="logCheck" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Log to the file:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
